### PR TITLE
Add cas_roles to permission scheme

### PR DESCRIPTION
**Pull request by rssk**

### CODE REVIEW

**Comment by cdaringe** [comment]
@rsskmr can you add us to the codeship perms, if possible?

**Comment by dylancwood** [comment]
@cdaringe FYI: the codeship tests will always fail until we figure out a way to mock our database. It's doable, but we aren't there yet.

**Comment by cdaringe** [comment]
@dylancwood roger that!

**Comment by dylancwood** [comment]
@rsskmr Why is this PR into `release` rather than `develop`?

File: lib/permissions/permission-schema.json [hunk 1]
```diff
@@ -1,26 +1,97 @@
...
+            "exists"
+        ],
+        "radletersallstudies": [
+            "exists"
+        ],
+        "querysuper": [
+            "exists"
+        ]
+    },
+    "meta": {
```
Comment (dylancwood) [inline lib/permissions/permission-schema.json:92]: consider naming this 'application', since it is an application-wide role?

**Comment by dylancwood** [comment]
Looks good. Let me know what you think about using `application` as the context for application-wide roles, instead of `meta`.

Also, please create a new PR that will merge into develop, rather than release.

**Comment by rssk** [comment]
Closing, was in release review mode, me bad

**Pull request closed without merging**